partials/contacts/favorites.html

```diff
@@ -1,20 +1,20 @@
 <div class="text-center" id="contacts-favorites">
     <h1>Favorites</h1>
     <div id="favorites">
-        <div ng-show="!favorites.length">
+        <div ng-show="!contacts.length">
```

Comment:
change this to use the 'ng-if' directive, since that directive will not put the html element into the dom *until* the condition is true.